### ISSUE TITLE
Wrong image size after cropping

### ISSUE THREAD

**Issue by budidino** [guoyingtao/Mantis#267]
Version: `2.6.0`

## Setup
```
var config = Mantis.Config()
config.cropViewConfig.cropShapeType = .circle(maskOnly: true)
config.presetFixedRatioType = .alwaysUsingOnePresetFixedRatio(ratio: 1/1)
```

## Issue
Selecting a screenshot image of size 1290 x 2796 with a crop ratio of `1:1` produces an image with the same resolution as the original, no matter which area or zoom I choose when cropping.

![Screenshot 2023-02-01 at 21 40 05](https://user-images.githubusercontent.com/1171894/216059567-29941f4b-620b-4a84-9188-d506bd4e9869.png)

**Comment by guoyingtao** [comment]
Hi @budidino 
Can you check if this issue is similar with #250?

**Comment by guoyingtao** [comment]
Does it happen to all images? Does it still work with 2.4.0 now?

**Comment by budidino** [comment]
> Hi @budidino Can you check if this issue is similar with #250?

It could be related, but I don't know if that is an iOS version issue. It seems like a vertical image issue. I tested it using `2.4.0` and `2.6.0`, and it is very easy to replicate on both... take a screenshot and use that photo for Mantis. It works ok with all my other images tested so far... mostly square or horizontal, but I wonder if that even matters.

**Comment by budidino** [comment]
> Does it happen to all images? Does it still work with 2.4.0 now?

Same issue on `2.4.0` and higher versions

**Comment by guoyingtao** [comment]
Can you check the Example project inside Mantis in master branch with your image and config?

**Comment by guoyingtao** [comment]
Would you mind posting one screenshot here so I can test with your screenshot? Thanks!

**Comment by guoyingtao** [comment]
Never mind. I reproduced it.

**Comment by guoyingtao** [comment]
@budidino 
I fixed this issue in this [fix-bitmapInfo-error branch](https://github.com/guoyingtao/Mantis/tree/fix-bitmapInfo-error)
Can you have a test to see if it works for you? Thanks!

**Comment by budidino** [comment]
I'll test it first thing tomorrow ;)

**Comment by guoyingtao** [comment]
I released Mantis 2.6.1 which includes this fix. @budidino
